### PR TITLE
Refactor: update humsci basic theme scss for namespaced consistency

### PR DESCRIPTION
**Pull request by kaseybon**
# READY FOR REVIEW

~⚠️ **DO NOT MERGE** until the following are merged:~ Safe to merge
- ✅STN-105
- ✅STN-106
- ✅STN-88
- ✅STN-109

## Summary
This pull request refactors our existing scss to conform our class/variable naming convention:
- We will continue to use BEM as it has been documented.
- We will no longer use class prefixes (`obj`, `util`, and `cmp`) as they only clutter the class names.
- We will namespace all classes, variables, functions (scss), and mixins with `.hb-`.

Documentation regarding namespacing has been added to the Humsci Basic `_main.scss`.

_Note: Right now this PR is dependent on the `STN-105--vertical-card-news` branch. Once that is merged I will update the PR base branch._

## Steps to Test
- [ ] Ensure that the CSS compiles without errors
- [ ] Run `npm test` and ensure that there are not scss linting or test errors
- [ ] Ensure that there are no style changes to the existing template (as compared to `STN-105--vertical-card-news`)
- [ ] Confirm that all Humsci Basic scss has been namespaced

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)

### CODE REVIEW

File: docroot/themes/humsci/humsci_basic/src/scss/components/_card.scss [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.hb-cmp-card {
+.hb-card {
```
Comment (silverli) [inline docroot/themes/humsci/humsci_basic/src/scss/components/_card.scss:1]: on line 95 and 113 there is still something referencing `cmp`

File: docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.general.scss [hunk 2]
```diff
@@ -1,3 +1,3 @@
-.hb-obj-page-width {
+.hb-page-width {
```
Comment (silverli) [inline docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.general.scss:1]: there is a reference to this in `docroot/themes/humsci/humsci_basic/templates/block/block--local-tasks-block.html.twig` that needs to be changed

**Review by silverli — CHANGES_REQUESTED** [review]
two small comments! looks good otherwise!

File: docroot/themes/humsci/humsci_basic/src/scss/_main.scss [hunk 3]
```diff
@@ -27,7 +29,23 @@
...
   'decanter/core/src/scss/components/local-footer',
 
-  // humsci_basic (ITSCSS)
+  // =====================================================================
+  // HUMSCI BASIC
+  // =====================================================================
+
+  // Namespacing
+  // ---------------------------------------------------------------------
+  // All classes, variables, and mixins that belong to the humsci_basic theme will
```
Comment (silverli) [inline docroot/themes/humsci/humsci_basic/src/scss/_main.scss:38]: 🎉

**Comment by bryanbraun** [comment]
I did a quick pass over the code and locally, and everything I saw looked good to me. Thanks for prioritizing this!

**Review by silverli — APPROVED** [review]
Looks awesome! All stylelint and sass tests still pass, and I don't see any non-prefixed variables or classes anymore.

**Pull request merged**